src/javascript/events/click.js

```diff
@@ -27,10 +27,12 @@ const eventPropertiesToCollect = [
 
 // Trigger the event tracking
 const track = eventData => {
-	const href = eventData.context.domPathTokens[0].href || null;
+	const firstDomPathToken = eventData.context.domPathTokens[0];
+	const href = firstDomPathToken.href || null;
+	const skipQueue = firstDomPathToken['data-o-tracking-skip-queue'] === 'true' || false;
```

Comment:
It might be worth adding a `.toLowerCase()` ... I'm not sure if that's already taken care of in the `domPathTokens` array.

``` javascript
const skipQueue = firstDomPathToken['data-o-tracking-skip-queue'].toLowerCase() === 'true' || false;
```

Side note: If you wanted to be a bit js-hipster about it, you could do this: (I think)
``` javascript
const skipQueue = !!(firstDomPathToken['data-o-tracking-skip-queue'] === 'true');
```

Reply:
I'd have to check to make sure `firstDomPathToken['data-o-tracking-skip-queue']` exists before I can use `toLowerCase()`, but I'll add that in.